@MikeNitsenko please check line length - 
```
tests/unit_tests/document_loaders/test_cube_semantic.py:40:89: E501 Line too long (139 > 88 characters)
tests/unit_tests/document_loaders/test_cube_semantic.py:50:89: E501 Line too long (156 > 88 characters)
```
I can take this up if you are offline; it may be late your time!

--- a/tests/unit_tests/document_loaders/test_cube_semantic.py
+++ b/tests/unit_tests/document_loaders/test_cube_semantic.py
@@ -37,7 +37,13 @@ class TestCubeSemanticLoader(TestCase):
 
         expected_docs: List[Document] = [
             Document(
-                page_content="table name: cube1, column name: sales, column data type: sum, column title: Sales, column description: None",
+                page_content=(
+                    "table name: cube1, "
+                    "column name: sales, "
+                    "column data type: sum, "
+                    "column title: Sales, "
+                    "column description: None"
+                ),
                 metadata={
                     "table_name": "cube1",
                     "column_name": "sales",
@@ -47,7 +53,13 @@ class TestCubeSemanticLoader(TestCase):
                 },
             ),
             Document(
-                page_content="table name: cube1, column name: product_name, column data type: string, column title: Product Name, column description: None",
+                page_content=(
+                    "table name: cube1, "
+                    "column name: product_name, "
+                    "column data type: string, "
+                    "column title: Product Name, "
+                    "column description: None"
+                ),
                 metadata={
                     "table_name": "cube1",
                     "column_name": "product_name",

--- a/tests/unit_tests/document_loaders/test_cube_semantic.py
+++ b/tests/unit_tests/document_loaders/test_cube_semantic.py
@@ -49,7 +49,7 @@ class TestCubeSemanticLoader(TestCase):
                     "column_name": "sales",
                     "column_data_type": "sum",
                     "column_title": "Sales",
-                    "column_description": None,
+                    "column_description": "None",
                 },
             ),
             Document(
@@ -65,7 +65,7 @@ class TestCubeSemanticLoader(TestCase):
                     "column_name": "product_name",
                     "column_data_type": "string",
                     "column_title": "Product Name",
-                    "column_description": None,
+                    "column_description": "None",
                 },
             ),
         ]

